.travis.yml

```diff
@@ -74,7 +74,7 @@ matrix:
     #  compiler: clang
 
     # test creating the manual
-    - env: TEST_SUITES=makemanuals
+    - env: TEST_SUITES=makemanuals NO_COVERAGE=1
```

Comment:
I assume the `NO_COVERAGE=1` here is only added for performance reasons? Regardless of that, a comment above would again be appreciated; it's already difficult enough to navigate and understand our `.travis.yml` :-)